message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -111,6 +111,16 @@
         <param index="6">Empty.</param>
         <param index="7">Empty.</param>
       </entry>
+      <entry value="42006" name="MAV_CMD_FIXED_MAG_CAL_YAW">
+        <description>Magnetometer calibration based on provided known yaw. This allows for fast calibration using WMM field tables in the vehicle, given only the known yaw of the vehicle. If Latitude and longitude are both zero then use the current vehicle location.</description>
+        <param index="1">YawDegrees.</param>
+        <param index="2">CompassMask, 0 for all.</param>
+        <param index="3">Latitude degrees.</param>
+        <param index="4">Longitude degrees.</param>
+        <param index="5">Empty.</param>
```

Comment:
Empty parameters should be left out

Reply:
I'd rather the empty be there.

Reply:
leaving in, as matches all the others in this file